### PR TITLE
[Lin Siyan]iP

### PR DESCRIPTION
**Pull request by Celineyaa**

### CODE REVIEW

File: src/main/java/Duke.java [hunk 1]
```diff
@@ -1,10 +1,61 @@
...
+
+            } else if (userInput.contains("mark")) {
+                if (userInput.contains("unmark")){
+
+                    indexOfMark = Integer.parseInt(userInput.split(" ")[1]) - 1;
+                    statusOfInput.set(indexOfMark, " ");
+                    System.out.println("OK, I've marked this task as not done yet:\n");
+                    System.out.println("[ ] " + userInputSaver.get(indexOfMark) + "\n");
+
+                }else{
```
Comment (annoy-o-mus) [inline src/main/java/Duke.java]: Lack of space surrounding the `else` keyword

File: src/main/java/Duke.java [hunk 2]
```diff
@@ -1,10 +1,61 @@
...
-    }
+        System.out.println("____________________________________________________________\n" + "Hello! I'm " + name
+                + "\n" + "What can I do for you?\n" + "\n"
+                + "____________________________________________________________\n");
+        Scanner scanner = new Scanner(System.in);
+        String userInput = scanner.nextLine();
+        userInput = userInput.toLowerCase();
+        List<String> userInputSaver = new ArrayList<>();
+        List<String> statusOfInput = new ArrayList<>();
+        while(!userInput.contains("bye")){
```
Comment (annoy-o-mus) [inline src/main/java/Duke.java]: Space missing after `while` keyword

File: src/main/java/Duke.java [hunk 3]
```diff
@@ -1,10 +1,61 @@
...
+
+            System.out.println("____________________________________________________________\n");
+            int indexOfMark;
+            if (userInput.contains("list")){
+                for (int i = 0; i < userInputSaver.size(); i++) {
+                    int inputIndex = userInputSaver.indexOf(userInputSaver.get(i))+1;
+                    System.out.println(inputIndex + "." + " [" + statusOfInput.get(i) + "] " + userInputSaver.get(i)+ '\n');
```
Comment (annoy-o-mus) [inline src/main/java/Duke.java]: Should wrap at 120 characters

File: src/main/java/Duke.java [hunk 4]
```diff
@@ -1,10 +1,61 @@
...
+                    System.out.println("OK, I've marked this task as not done yet:\n");
+                    System.out.println("[ ] " + userInputSaver.get(indexOfMark) + "\n");
+
+                }else{
+                    indexOfMark = Integer.parseInt(userInput.split(" ")[1]) - 1;
+                    statusOfInput.set(indexOfMark, "X");
+                    System.out.println("Nice! I've marked this task as done:\n");
+                    System.out.println("[X] " + userInputSaver.get(indexOfMark) + "\n");
+                }
+            }else{
```
Comment (annoy-o-mus) [inline src/main/java/Duke.java]: Lack of space surrounding `else` keyword

File: src/main/java/Duke.java [hunk 5]
```diff
@@ -1,10 +1,61 @@
...
+        Scanner scanner = new Scanner(System.in);
+        String userInput = scanner.nextLine();
+        userInput = userInput.toLowerCase();
+        List<String> userInputSaver = new ArrayList<>();
+        List<String> statusOfInput = new ArrayList<>();
+        while(!userInput.contains("bye")){
+
+            System.out.println("____________________________________________________________\n");
+            int indexOfMark;
+            if (userInput.contains("list")){
```
Comment (annoy-o-mus) [inline src/main/java/Duke.java]: missing spaces before opening brace

**Comment by Celineyaa** [comment]
Dear sir/madam,

Sorry to bother, however I tried to PR my master branch after I finished the level 1 2 and 3, but I can't find my pushed changes on my remote master branch, the updates only show up at the ipMyAnswer branch, however when I pushed in SourceTree, I chose both branches to push. So is there any way for me to show my updates on my master branches and send PR using my master branch? I will be grateful if you can do me this favor!

Yours,
Siyan

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: nus-se-bot ***@***.***>
发送时间: Thursday, February 8, 2024 11:23:17 AM
收件人: nus-cs2113-AY2324S2/ip ***@***.***>
抄送: LIN Siyan ***@***.***>; Mention ***@***.***>
主题: Re: [nus-cs2113-AY2324S2/ip] [Lin Siyan]iP (PR #186)


@Celineyaa<https://github.com/Celineyaa> It looks like you sent this PR to the wrong repo. Only one PR needs to be sent to this upstream repo.
Closing this PR ...

―

**Pull request closed without merging**